thomaspoignant/go-feature-flag
# fix: unassign issue workflow

## Description

- Problem:
Assignee can take issue using `onlydustapp` bot, and this bot does not assign `Assigned` label
We need to check stale issue based on assignee list
Reference: https://github.com/actions/stale?tab=readme-ov-file#include-only-assigned
- The way it was resolved:
Change stale issue check conditions

- How to test the change:
Change workflow schedule (`cron`), number of days before being marked as stale (`env.DAYS_UNTIL_STALE`) in `unassign-issue.yml`, and check

## Closes issue(s)

Resolve #2313

## Checklist
